modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -264,17 +263,13 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
   }
 
   /**
-   * Signs the transaction associated to the transaction request.
+   * Gets signing key, txRequestResolved and txRequestId
    * @param {string | TxRequest} params.txRequest - transaction request object or id
    * @param {string} params.prv - decrypted private key
    * @param { string} params.reqId - request id
-   * @returns {Promise<TxRequest>} fully signed TxRequest object
+   * @returns {@Promise<ECDSASigningRequestBaseResult>}
```

Comment:
I am not too sure if the `@` is necessary + the jsdoc `@returns` type is different from the actual function return type